request_handler.go

```
@@ -127,58 +94,115 @@ func readFromPublic(username string) (*PublicInfo, error) {
 	return &pageData, err
 }
 
+//
 func accountsHandler(w http.ResponseWriter, r *http.Request, title string) {
 	p, err := readFromPublic(title)
 	if err != nil {
 		log.Println(err)
 	}
+
 	page := Profile{}
 	page.Username = p.Username
 	page.Description = p.Description
 	renderTemplate(w, "public_profile", &page)
 }
 
 // Render the edit information page
-
 func editHandler(w http.ResponseWriter, r *http.Request, title string) {
-	p, err := readFromBackend(title)
+	p, err := readFromBackend(title,r)
 	if err != nil {
 		log.Println(err)
```

It shouldn't just swallow the error.